jupyterhub/apihandlers/users.py

```diff
@@ -168,7 +172,9 @@ def patch(self, name):
         for key, value in data.items():
```

Comment:
This is where auth_state would need special handling:

```python
if 'auth_state' in data:
    auth_state = data.pop('auth_state')
    await user.set_auth_state(...
```

Reply:
Nods